shared/fs/really-delete/container.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+import * as FsGen from '../../actions/fs-gen'
+import * as RouteTreeGen from '../../actions/route-tree-gen'
+import * as Container from '../../util/container'
+import ReallyDelete from '.'
+import {anyWaiting} from '../../constants/waiting'
+import * as Types from '../../constants/types/fs'
+
+type OwnProps = Container.RouteProps<{path: Types.Path}, {}>
+
+const mapStateToProps = (state, ownProps) => {
+  const path = Container.getRouteProps(ownProps, 'path')
+  return {
+    _deleting: anyWaiting(state, Types.deleteFolderWaitingKey(path)),
+    path: path,
+    title: 'Confirmation',
+  }
+}
+
+const mapDispatchToProps = (dispatch, ownProps) => ({
+  onBack: () => dispatch(RouteTreeGen.createNavigateUp()),
+  onDelete: () => {
+    const path = Container.getRouteProps(ownProps, 'path')
+    dispatch(FsGen.createDeleteFile({path}))
+    // TODO: I think I should be closing this using the waiting key instead,
+    //  but I'm not sure how that works.
+    dispatch(RouteTreeGen.createNavigateUp())
```

Comment:
I don't think this is relevant for nav2. @songgao what's the right way to navigate out of a menu in nav2?

Reply:
I think this should work; there's also a way to clear all modals if you have more than one layer, but this is just one layer so navigateUp works too.